tools/tools.go

```diff
@@ -86,6 +90,42 @@ func Sed(oldValue, newValue, filePath string) error {
 	return err
 }
 
+func SshRun(userName, userPassword, server, cmd string) (string, error) {
+	// Define ssh connection
+	config := &ssh.ClientConfig{
+		User: userName,
+		Auth: []ssh.AuthMethod{
+			ssh.Password(userPassword),
+		},
+		HostKeyCallback: ssh.InsecureIgnoreHostKey(),
+	}
+
+	client, err := ssh.Dial("tcp", server+":22", config)
+	if err != nil {
+		// Failed to dial
+		return "", err
+	}
+
+	// Open a client session
+	session, err := client.NewSession()
+	if err != nil {
+		// Failed to create session
+		return "", err
+	}
+	defer session.Close()
+
+	// Execute the command on the opened session
+	var stdout bytes.Buffer
+	var stderr bytes.Buffer
+	session.Stdout = &stdout
+	session.Stderr = &stderr
+	if err := session.Run(cmd); err != nil {
```

Comment:
nit: maybe we should log command being run? Or is that done somewhere else?

Reply:
Not done because this function will be used mainly in test code. So yes I can either log in it or it can be done in the test code if needed. In my case I don't need to log, I just need to be sure that the ssh connection is done and working.